src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -202,6 +186,30 @@ private async Task<string> GetMetadataSchema(Uri url)
             };
         }
 
+        private async Task<string> FetchRetry(string url, IReadOnlyDictionary<string, string> headers = null, bool nullOn404 = false)
+        {
+            try
+            {
+                using var response = await HttpPolicyExtensions
+                   .HandleTransientHttpError()
+                   .Or<OperationCanceledException>()
+                   .Or<IOException>()
+                   .RetryAsync(3, onRetry: (_, i) => Log.Write($"[{i}] Retrying '{url}'"))
+                   .ExecuteAsync(async () =>
+                   {
+                       var content = await Fetch(url, headers);
```

Comment:
Consider use `_http.SendAsync` directly here and move line 229 - 232 here as that one is validation specific.

Reply:
Done

Reply:
I didn't see anything changed here, did you forget to push?

Reply:
me either

Reply:
There was an issue with my desktop to push code, and now it is fixed. Sorry for confusion.